api/src/main/java/org/openmrs/hl7/handler/ORUR01Handler.java

```diff
@@ -358,10 +358,9 @@ private Message processORU_R01(ORU_R01 oru) throws HL7Exception {
 				}
 				finally {
 					// Handle obs-level exceptions
-					if (errorInHL7Queue != null) {
-						throw new HL7Exception(Context.getMessageSourceService().getMessage(
```

Comment:
I think this should be caught in the `catch` block. cc @dkayiwa

Reply:
@f4ww4z shall I do it or wait for @dkayiwa to reply

Reply:
Actually, I have not created this PR with git so how can I now squash the commits

Reply:
squashed them